HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,69 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+
+log2(inf.asInteger)
+2 ** 31 - inf.asInteger
+
+subsection::Using Floats as replacement for Integers
+
+In SuperCollider, Floats are 64 bit wide. Because  an link::Classes/Integer:: is only 32 bit, it can capture only numbers up to code::± 2147483647::, which is code::2^31:: (one bit is for the sign), or about code::2 x 10^9::.
+
+Therefore, in some situations it can be useful to calculate with floats also when only whole numbers are needed. You can use 64 bit floats for integer calculations up to code::± 9007199254740992:: (code::2^53::, or about code::9 x 10^15::). Note that you still may get floating point errors when you apply certain operations.
```

Comment:
better to say "integers" or "integral numbers"; "whole numbers" are the positive integers

also in general the convention is to write "32-bit" and "64-bit" with hyphens.

Reply:
what does the last sentence here mean? could it be made more descriptive perhaps?

Reply:
I've taken some time to find a counterexample, where it fails to match with this precision, but I couldn't. If I find one, I'll add it, but for now I've removed the comment.